pkg/config/config.go

```diff
@@ -1067,10 +1067,13 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("admission_controller.cws_instrumentation.command_endpoint", "/inject-command-cws")
 	config.BindEnvAndSetDefault("admission_controller.cws_instrumentation.target.namespaces", []string{})
 	config.BindEnvAndSetDefault("admission_controller.cws_instrumentation.target.all_namespaces", true)
+	config.BindEnvAndSetDefault("admission_controller.cws_instrumentation.target.skip_namespaces", []string{})
 	config.BindEnvAndSetDefault("admission_controller.cws_instrumentation.mutate_unlabelled", false)
 	config.BindEnvAndSetDefault("admission_controller.cws_instrumentation.cws_injector_container_registry", "")
 	config.BindEnvAndSetDefault("admission_controller.cws_instrumentation.cws_injector_image_name", "datadog/cws-injector")
 	config.BindEnvAndSetDefault("admission_controller.cws_instrumentation.cws_injector_image_tag", "master")
+	config.BindEnv("admission_controller.cws_instrumentation.init_resources.cpu")
```

Comment:
I think it would make sense to have a default value here, my understanding is that it's not workload specific

Reply:
Discussed offline with @clamoriniere, we're going to add default values in our helm charts 👍🏻